src/collections/_documentation/workflow/releases/release-automation/jenkins/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Jenkins
+sidebar_order: 0
+
+---
+
+This guide walks you through the process of automating Sentry release management and deploy notifications in Jenkins. After deploying in Jenkins, you'll be able to identify suspect commits that are likely the culprit for new errors. You'll also be able to apply source maps to see the original code in Sentry.
+
+Before starting, confirm that your Sentry project is properly set up to track commit metadata. The easiest method to do so is to [install a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration). Once that's installed, and you've added your repository, come back to this guide. If you've already installed a repository integration, you're ready to go.
+
+## Create a Sentry Internal Integration
+
+For Jenkins to communicate securely with Sentry, you'll need to create a new internal integration. In Sentry, navigate to **Settings > Developer Settings > New Internal Integration**.
+
+Give your new integration a name (for example, “Jenkins Deploy Integration”) and specify the necessary permissions. In this case, we need Admin access for “Release” and Read access for “Organization”.
+
+See our [docs page on permissions]({%- link _documentation/api/permissions.md -%}) to learn more about scopes for Sentry’s API endpoints.
+
+[{% asset releases/jenkins/internal-integration-permissions.png alt="View of internal integration permissions." %}]({% asset releases/jenkins/internal-integration-permissions.png @path %})
+
+Click “Save” at the bottom of the page and grab your token, which you'll need this in the next step.
+
+## Add Token to Jenkins Credentials
+
+To access our internal integration token securely in Jenkins, store it as a [credential](https://www.jenkins.io/doc/book/using/using-credentials/):
+
+1. From the Jenkins home page (the Dashboard of the classic Jenkins UI), click **Credentials > System**.
+2. Under **System**, click the **Global credentials (unrestricted)** link to access this default domain.
+3. Click **Add Credentials**.
+4. From the **Kind** field select Secret Text.
+5. Give your credential an **ID** (for example, "sentry-auth-token") and in the **Secret** field paste your internal integration token.
+6. Click **OK** to save the credentials.
+
+[{% asset releases/jenkins/jenkins-add-credential.png alt="View of Jenkins add credentials." %}]({% asset releases/jenkins/jenkins-add-credential.png @path %})
+
+
+## Notify Sentry of Deployment
+
+Depending on what type of Jenkins project you are using to build and deploy, you want to follow different steps:
```

Comment:
Updating to make the contraction consistent with the rest of docs and this page.
```suggestion
Depending on what type of Jenkins project you're using to build and deploy, you want to follow different steps:
```